docs/articles/infrastructure/ci-cd/jenkins.rst

```diff
@@ -19,7 +19,7 @@ The Jenkins project offers preinstalled Docker images which can be used to vastl
    TimeoutStartSec=0
    Restart=always
    ExecStartPre=-/usr/bin/docker stop %n
-   ExecStartPre=-/usr/bin/docker rm %n
+#   ExecStartPre=-/usr/bin/docker rm %n
```

Comment:
This line should be indented so that this is correctly generated as a code block.

```suggestion
    #   ExecStartPre=-/usr/bin/docker rm %n
```